pkg/apis/v1alpha1/serviceexport.go

```diff
@@ -88,3 +92,134 @@ type ServiceExportList struct {
 	// +listType=set
 	Items []ServiceExport `json:"items"`
 }
+
+const (
+	// ServiceExportConditionAccepted is true when the Service Export is accepted.
+	// This does not indicate whether or not the configuration has been exported
+	// to a control plane / data plane.
+	//
+	//
+	// Possible reasons for this condition to be true are:
+	//
+	// * "Accepted"
+	//
+	// Possible reasons for this condition to be False are:
+	//
+	// * "NoService"
+	// * "InvalidServiceType"
+	//
+	// Controllers may raise this condition with other reasons,
+	// but should prefer to use the reasons listed above to improve
+	// interoperability.
+	ServiceExportConditionAccepted = "Accepted"
+
+	// ServiceExportReasonAccepted is used with the "Accepted" condition when the
+	// condition is True.
+	ServiceExportReasonAccepted = "Accepted"
+
+	// ServiceExportReasonNoService is used with the "Accepted" condition when
+	// the associated Service does not exist.
+	ServiceExportReasonNoService = "NoService"
+
+	// ServiceExportReasonInvalidServiceType is used with the "Accepted"
+	// condition when the associated Service has an invalid type
+	// (per the KEP at least the ExternalName type).
+	ServiceExportReasonInvalidServiceType = "InvalidServiceType"
+)
+
+const (
+	// ServiceExportConditionExported is true when the service is exported to some
+	// control plane or data plane.
+	//
+	//
+	// Possible reasons for this condition to be true are:
+	//
+	// * "Exported"
+	//
+	// Possible reasons for this condition to be False are:
+	//
+	// * "Pending"
+	// * "Failed"
+	//
+	// Possible reasons for this condition to be Unknown are:
+	//
+	// * "Pending"
+	//
+	// Controllers may raise this condition with other reasons,
+	// but should prefer to use the reasons listed above to improve
+	// interoperability.
+	ServiceExportConditionExported = "Exported"
+
+	// ServiceExportReasonExported is used with the "Exported" condition
+	// when the condition is True.
+	ServiceExportReasonExported = "Exported"
+
+	// ServiceExportReasonPending is used with the "Exported" condition
+	// when the service is going to be exported.
+	ServiceExportReasonPending = "Pending"
+
+	// ServiceExportReasonFailed is used with the "Exported" condition
+	// when the service failed to be exported.
+	ServiceExportReasonFailed = "Failed"
+)
+
+const (
+	// ServiceExportConditionConflicted indicates that the controller was unable
+	// to resolve conflict for a ServiceExport. This condition must be at
+	// least raised on the conflicting ServiceExport and is recommended to
+	// be raised on all on all the constituent `ServiceExport`s if feasible.
```

Comment:
It indicates there was some conflicting aspect between 2 or more exported services but it doesn't mean it wasn't resolved. Also I'm not sure the second statement re: where the condition should be raised is really relevant here.

```suggestion
	// ServiceExportConditionConflicted indicates that some property of an exported service has conflicting 
        //  values across the constituent ServiceExports.
```